opencensus/common/BUILD

```diff
@@ -0,0 +1,29 @@
+# OpenCensus C++ Common library.
+# See ../../README.md for details.
+#
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//opencensus:copts.bzl", "DEFAULT_COPTS", "TEST_COPTS")
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:private"])
+
+cc_library(
+    name = "common",
```

Comment:
I'd prefer name = "version"

Reply:
This follows your convention in all the packages that you have only one artifact with the name of the package. If named version probably should not be public.

Reply:
Let's keep `name = "common"` but mark it private unless and until there's a compelling reason to expose it.
 
I was hoping to leave //opencensus/common:common for some kind of utility library later. :)

Reply:
Changed to version and made it package visible only.